doc/source/security/encrypting-tables.rst

```diff
@@ -0,0 +1,49 @@
+.. _encrypting-tables:
+
+=========================================================
+Encrypting File-Per-Tablespace Tables
+=========================================================
+
+When you define the variable ``innodb_file_per_table`` in the my.cnf file, InnoDB uses a tablespace file for each InnoDB table and creates and stores the table data and the indexes in a single data file. In this tablespace configuration, each table is stored in an .ibd file.
+
+You can encrypt InnoDB tables stored in ``innodb_file_per_table`` tablespaces.
+
+The architecture for data at rest encryption is two tier:
+
+* Master key
+* Tablespace keys.
+
+For encryption, you must have the keyring plugin installed and enabled.
+
+You can set the `innodb_file_per_table` in my.cnf, or start the server with the ``--innodb_file_per_table`` option. With the configuration change, you must restart the server.
```

Comment:
This is default - so no need to specify innodb_file_per_table explicitly.

Reply:
removed and rewritten.